packages/cli/src/commands/keys/add.ts

```diff
@@ -45,7 +45,7 @@ Uploading SSH public key /my/key.pub... done`
   }
 
   static args = {
-    key: Args.string(),
+    key: Args.string({description: 'The absolute path to the key located on disk. If omitted, the default rsa key is used.'}),
```

Comment:
Change to "absolute path to the key located on disk. If omitted, we use the default rsa key."